src/helm/benchmark/window_services/test_t0pp_window_service.py

```diff
@@ -87,62 +87,62 @@ def test_decode(self):
 
     def test_tokenize(self):
         assert self.window_service.tokenize(TEST_PROMPT) == [
-            "▁The",
-            "▁Center",
-            "▁for",
-            "▁Research",
-            "▁on",
-            "▁Foundation",
-            "▁Model",
+            "The",
```

Comment:
Could we exclude the `_` tokenizers from this PR? You might have to add a branch in huggingface_client to do that i.e. special-case GPT-2...

This strips the leading spaces from tokens, which is possibly worse. This means that if you concat the tokens, you get "TheCenterforResearch...". Also it modifies the byte count and thus the metrics.

I haven't tested this yet, but I expect that: `convert_ids_to_string([37, 1166])` gives "The Center" (with space) and `convert_ids_to_string([1166])` alone gives "Center" (without space).

Reply:
Ok I added a special case. This is not super clean but it should work. Regarding the `_` what should I do? Replace it manually?